test/e2e/multihoming.go

```diff
@@ -792,6 +792,95 @@ var _ = Describe("Multi Homing", func() {
 						Expect(connectToServer(clientPodConfig, underlayServiceIP, servicePort)).To(MatchError(ContainSubstring("Connection timeout")))
 					})
 				})
+
+				When("an ingress deny-all policy is provisioned", func() {
+					var clientPodConfig podConfiguration
+					allPodsSelector := map[string]string{}
+
+					const (
+						denyAllIngress = "deny-all-ingress"
+						allowAllEgress = "allow-all-egress"
+						denyAllEgress  = "deny-all-egress"
+					)
+
+					BeforeEach(func() {
+						clientPodConfig = podConfiguration{
+							name:        clientPodName,
+							namespace:   f.Namespace.Name,
+							attachments: []nadapi.NetworkSelectionElement{{Name: secondaryNetworkName}},
+							labels:      allPodsSelector,
+						}
+						kickstartPod(cs, clientPodConfig)
+					})
```

Comment:
Move all this to the DescribeTable function to simplify the test.

Reply:
Assuming you mean to move the code in the BeforeEach to the DescribeTable body,
I think @maiqueb was deliberately done like that
(as we discussed it offline)

wdyt ?

Reply:
> Move all this to the DescribeTable function to simplify the test.

How is that simplifying ?

Reply:
> > Move all this to the DescribeTable function to simplify the test.
> 
> How is that simplifying ?

Reducing nesting at ginkgo always simplify test description and it make easier to follow the flow, if the BeforeEach is used by just one DescribeTable it's not worth it to keep it.

Reply:
but it also removes mental burden that the test does only what it need, and the BeforeEach is the preparation

Reply:
but at least change this  `an ingress deny-all policy is provisioned`.

Reply:
Done